assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -149,12 +193,111 @@ const baseActions = {
 
 		return { response, error };
 	},
+
+	/**
+	 * Registers a module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string}    slug                         Module slug.
+	 * @param {Object}    [settings]                   Optional. Module settings.
+	 * @param {string}    [settings.name]              Optional. Module name. Default is the slug.
+	 * @param {string}    [settings.description]       Optional. Module description. Default empty string.
+	 * @param {string}    [settings.icon]              Optional. Module icon. Default empty string.
+	 * @param {number}    [settings.order]             Optional. Numeric indicator for module order. Default 10.
+	 * @param {string}    [settings.homepage]          Optional. Module homepage URL. Default empty string.
+	 * @param {WPElement} [settings.settingsComponent] React component to render the settings panel. Default is the DefaultModuleSettings component.
+	 * @return {Object} Redux-style action.
+	 */
+	*registerModule( slug, { settingsComponent = DefaultModuleSettings, ...settings } = {} ) {
+		invariant( slug, 'module slug is required' );
+
+		const registry = yield commonActions.getRegistry();
+		yield actions.waitForModules();
+		registry.select( STORE_NAME ).getModules();
```

Comment:
What is the reason for this call to this selector? Its return value isn't being stored anywhere.